docs/src/scs/sbom.md

```diff
@@ -0,0 +1,34 @@
+# Software bill of materials (SBOM)
```

Comment:
Please use the title case for all headings in the file.
https://github.com/tiiuae/ghaf/blob/main/docs/style_guide.md#headings

Reply:
Resolved in https://github.com/tiiuae/ghaf/pull/46/commits/50a79b10dcbefd55fef9dfd3a7dacf8e4a1fafc7